docs/Developers/Writing_Good_Commits.rst

```diff
@@ -161,6 +161,17 @@ Example 2 (implemented feature):
 
     Closes https://github.com/coala/coala/issues/2060
 
+Example 3 (fixed typo):
+
+::
+
+    configure.py: Fix docstring typo
```

Comment:
Since this is a real example taken from https://github.com/coala/coala/pull/4024 , you should also use the real file `ConsoleInteraction.py`